modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -2596,6 +2597,16 @@ export class Wallet implements IWallet {
     return new Lightning(this.bitgo, this);
   }
 
+  /**
+   * Create inscriptions for btc/tbtc from this wallet
+   */
+  public inscription(): Inscription {
```

Comment:
I know we're trying to keep up with the `lightning` convention here, but using only `inscription` doesn't clearly let the reader know what the function is doing, which is to create an Inscription instance. Can we rename it to `getInscriptionInstance` or `createInscriptionInstance`?

Reply:
Hmm personally prefer just inscription(). Lets vote on this on our channel.

Reply:
updated to createInscriptionInstance